packages/trader/src/AppV2/Components/Guide/Description/video-fragment.tsx

```diff
@@ -31,22 +42,16 @@ const VideoFragment = ({ contract_type }: TVideoFragment) => {
             })}
         >
             {is_loading && <Loading is_fullscreen={false} />}
-            <video
-                autoPlay
-                className='video-fragment'
-                data-testid='dt_video_fragment'
+            <DotLottieReact
+                autoplay
+                dotLottieRefCallback={
```

Comment:
Unfortunately, we can't add onLoadedData directly to this component. According to documentation, DotLottieReact component can receive a dotLottieRefCallback prop that can be used to get a reference to the [dotLottie](https://github.com/LottieFiles/dotlottie-web/blob/HEAD/packages/web/README.md#apis) web player instance. This reference can be used to listen to player events emitted by the [dotLottie](https://github.com/LottieFiles/dotlottie-web/blob/HEAD/packages/web/README.md#events) web instance.

Reply:
Also, we can't store `dotLottie` in `ref.current` as in this case` React.useEffect` won't recognize it as a dependency.

Reply:
sure, it's fine as this approach follows the official doc: https://github.com/LottieFiles/dotlottie-web/blob/main/packages/react/README.md
Thank you)